pipeline/viralrecon.config

```diff
@@ -5,3 +5,4 @@
  */
 
 includeConfig "${params.custom_config_base}/conf/pipeline/viralrecon/genomes.config"
+includeConfig "${params.custom_config_base}/conf/pipeline/viralrecon/eddie.config"
```

Comment:
```suggestion

profiles {
  eddie { includeConfig "${params.custom_config_base}/conf/pipeline/viralrecon/eddie.config" }
}
```

Reply:
The `genomes.config` always needs to be loaded when running the pipeline which is why it is outside of the `profiles` block.